docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     - PYTHONPATH=./openstates
     - NEW_YORK_API_KEY
     - INDIANA_API_KEY
+    - VA_USER
+    - VA_PASSWD
```

Comment:
to bring these in line with the other environment variables for clarity can we rename these to

VIRGINIA_FTP_USER
VIRGINIA_FTP_PASSWORD

Reply:
Resolved.